pallets/account-manager/Cargo.toml

```diff
@@ -60,5 +60,6 @@ std = [
   "t3rn-primitives/std",
   "pallet-xbi-portal/std",
   "circuit-runtime-types/std",
+  "circuit-runtime-pallets/std",
```

Comment:
This would bring in the dev dependency for std when it isn't brought into the binary. Only things in the `dependencies` table should go here.